lib/commons/aria/index.js

```diff
@@ -189,7 +189,7 @@ lookupTables.role = {
 		context: null
 	},
 	'application': {
-		type: 'landmark',
```

Comment:
This is causing the following test to fail, be sure to fix that:

`1) table.isDataTable should be true if the table has a landmark role application:`

Reply:
@WilcoFiers 
In ARIA 1.1, 'application' is not a landmark role.
https://www.w3.org/TR/wai-aria-1.1/#application
The new superclass role is 'structure'. 
Can you update the 'develop' branch to meet the ARIA 1.1 specification?